portal/views/auth.py

```diff
@@ -50,6 +52,329 @@
 
 auth = Blueprint('auth', __name__)
 
+google_blueprint = make_google_blueprint(
+    scope=['profile', 'email'],
+    login_url='/login/google/',
+)
+
+facebook_blueprint = make_facebook_blueprint(
+    scope=['email', 'user_birthday', 'user_gender'],
+    login_url='/login/facebook/',
+)
+
+
+class FlaskDanceProvider:
```

Comment:
@pbugni where is the best place to put these classes? Should I leave them in this file or put them in their own file(s)/folder(s). I left them here because it's consistent with other files, but this file is getting pretty long.